src/lib/structures/game/BinbGame.ts

```diff
@@ -168,32 +166,15 @@ export class BinbGame extends Game {
 		const row = new MessageActionRow() //
 			.addComponents(
 				new MessageButton() //
-					.setCustomId(CustomIds.Join + voiceChannelMembers.first()!.guild.id)
+					.setCustomId(`${CustomIds.Join}|${voiceChannelMembers.first()!.guild.id}`)
```

Comment:
Can't you just do this?
```suggestion
					.setCustomId(`${CustomIds.Join}|${this.guild.id}`)
```